ruby/spec/pehape/string/echo_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+RSpec.describe Pehape do
+  it "echo all type data" do
+    expect(Pehape.echo("Pe ha pe")).to equal(nil)
```

Comment:
assert stdoutnya, jangan return valuenya
http://rspec.info/blog/2014/05/notable-changes-in-rspec-3/#new-output-matcher